.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: go
 sudo: required
+group: deprecated-2017Q2
```

Comment:
With https://github.com/mattes/migrate/pull/262 merged, this is no longer necessary, and will not work starting in September.

You should pull/rebase to include that commit and remove this line 🙂